src/app/containers/Include/index.canonical.jsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import { string } from 'prop-types';
+import { Helmet } from 'react-helmet';
+import styled from 'styled-components';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
```

Comment:
Could we use `@bbc/psammead-grid` here.

Reply:
we could but I'd suggest that piece of work should be isolated in an another issue as we'll be doing the same refactor to the canonical implementation